src/pynecore/cli/commands/run.py

```diff
@@ -85,6 +86,10 @@ def run(
                                      help="PyneSys API key for compilation (overrides configuration file)",
                                      envvar="PYNESYS_API_KEY",
                                      rich_help_panel="Compilation Options"),
+        symbol: str | None = Option(None, "--symbol", "-s",
```

Comment:
We don't need symbol on run. However we may try to use the filename by default as symbol name.